Week-14/Link_List_1/merge_two_sorted.js

```diff
@@ -0,0 +1,23 @@
+function mergeTwolist(list1, list2) {
```

Comment:
[mandatory]: declare the linkedlist class

Reply:
I have declared the class